### DISCUSSION TITLE
Chrome extensions support?

### DISCUSSION THREAD

**Discussion by tferr** [fjwillemsen/NativeOverleaf#19] [Ideas]
Thanks for this tool @fjwillemsen. Super useful.
Do you know how one could use [SLext](https://github.com/hrjakobsen/slext) from within? 
SLext's has a super useful command bar, that I use quite frequently.

**Comment by fjwillemsen** [comment]
Thank you! It's definitely on my to do list, though I have not yet found a good method to do this with Nativefier, and extensions should not collide with the changes made by Native Overleaf. Let this be the official issue to discuss this. Does anyone know a way to do this with Nativefier?

**Comment by tferr** [comment]
Some of this has been discussed here: https://github.com/nativefier/nativefier/issues/207. I guess, it is not that trivial ? 
Also, this project seems relevant: https://github.com/samuelmaddock/electron-browser-shell/tree/master/packages/electron-chrome-extensions

**Comment by nanbaima** [comment]
I suggest the integration of Writefull. As we also use this extension in our University.
But I have to say that the OS Grammarly works just fine.
Maybe this could be done via extensions or via API maybe?
Oooww, maybe having ChatGPT API also could be something, just putting it out there. 😅

**Comment by fjwillemsen** [comment]
Apologies for not getting back to this earlier. I have looked into adding extensions, but unsuccessfully so far. I will convert this discussion to an issue and request community help in the hopes that someone else can look into it.
